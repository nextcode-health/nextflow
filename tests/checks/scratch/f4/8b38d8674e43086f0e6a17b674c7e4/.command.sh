#!/bin/bash -ue
echo 'x: 2; y: Hola; z: beta'
