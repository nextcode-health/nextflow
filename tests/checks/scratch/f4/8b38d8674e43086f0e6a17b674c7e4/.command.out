x: 2; y: Hola; z: beta
